From 5980950dbae82c6a03b38b8e66e07a8b29068a00 Mon Sep 17 00:00:00 2001
From: AsavarTzeth <asavartzeth@gmail.com>
Date: Thu, 19 Jul 2018 21:55:38 +0200
Subject: [PATCH] Fix makefile

- ldconfig should not be called by Makefile
- Add missing symlink for major version of lib
---
 libs/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libs/Makefile b/libs/Makefile
index 186982d..1ca0f1c 100644
--- a/libs/Makefile
+++ b/libs/Makefile
@@ -34,7 +34,6 @@ CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
 CPPFLAGS += -DENABLE_VECTOR_MODE 
 CXXFLAGS += -fPIC -Wall
 CXXFLAGS += -ffast-math -funroll-loops -O3
-CXXFLAGS += -march=native
 LDLFAGS += 
 LDLIBS +=
 
@@ -57,7 +56,7 @@ install:	$(ZITA-CONVOLVER_MIN)
 	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(PREFIX)/include
 	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)
 	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-CONVOLVER_SO)
-	ldconfig
+	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-CONVOLVER_MAJ)
 
 clean:
 	/bin/rm -f *~ *.o *.a *.d *.so.*
-- 
2.18.0

